CSharp/BatchExplorer/BatchExplorer.csproj

```diff
@@ -425,6 +427,24 @@
   <ItemGroup>
     <Resource Include="readme.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <Resource Include="Images\add.png" />
+    <Resource Include="Images\delete.png" />
+    <Resource Include="Images\refresh.png" />
```

Comment:
We should think about if we can avoid having .png's in the repo directly but not sure how we should accomplish that.